
vector<ObjectState> env_state_vec_from_container